holmes/plugins/toolsets/teams/teams.py

```
@@ -0,0 +1,446 @@
+        if chat_type == "group" and len(member_ids) < 2:
+            return StructuredToolResult(
+                status=StructuredToolResultStatus.ERROR,
+                error="group chats require at least 2 member ids in addition to the authenticated user.",
+                params=params,
+            )
+
+        # Resolve the authenticated user's id so we can add ourselves explicitly.
+        try:
+            me, _ = self._toolset._graph_request("GET", "/me", timeout=10)
+            my_id = me["id"]
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)